docs/examples/numba_func/Numba_func.ipynb

```diff
@@ -0,0 +1,374 @@
+{
```

Comment:
<p>I'm not sure if it won't be easier to pass the types and ndims in the pipeline definition itself. Either way, please explain it why we do it and what the passed values mean. Probably mention that we do not adjust ndim for batch mode.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2886/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>